data/scenarios/questions.yaml

```diff
@@ -0,0 +1,108 @@
+#
+#                       -- data requirements --
+#       data_sources        A list of data sources (as defined in the
+#                           data_finder.yaml configuration file) that
+#                           either need to be present, should not be
+#                           present or are optional in the dataset in
+#                           order to be able to answer the question.
+#
+#       parameters          A list of parameters that will be asked
+#                           in the investigation, to provide context
+#                           needed for the question. This can be
+#                           simple things like the username of the user
+#                           the analyst is looking at, or a domain that
+#                           they are interested in, that is some context
+#                           that is needed in order for the question to be
+#                           properly asked.
+#
+#                       -- context/enrichment --
+#       analyzers           A list of analyzers that will be run on the
+#                           dataset.
+#
+#       graphs              A list of graph plugins that will assist the
+#                           analyst with answering the question.
+#
+#       search_templates    A list of search templates that will assist the
+#                           analyst with answering the question.
+#
+#                       -- answers --
+#       answer_simple       If the answer can be simply derived from a search
+#                           in the dataset, an answer source can be defined.
+#                           It contains two section, a "must" or "must_not",
+#                           which is a list of data finder defintions that all
+#                           must or must not return True values. This can be
+#                           used to define things that should be present in the
+#                           dataset or should not be there in order to give
+#                           an answer to the investigative question.
+#
+#       answer_analyzer     If the answer cannot be derived from a simple
+#                           search a name of an analyzer can be provided
+#                           that is specifically designed to answer this
+#                           particular investigative question.
+# ------------------------------------------------------------------------
+user_logged_in_win:
+  display_name: Did someone log into a Windows machine?
+  description: This question aim to answer whether
+    someone (anyone) logged into the machine. It will run the login analyzer
+    to determine if there was any login activity. This question only
+    looks at activity logged in the EVTX file, therefore it only
+    applies to Windows machines.
+  data_sources:
+    - windows_login
+  analyzers:
+    - login
+  answer_simple:
+    must:
+      - windows_login
+
+user_downloaded_file:
```

Comment:
Would it make sense to first answer the question "Did someone download files" and if that question is answered, we move on the more scoped one?

Reply:
Yes, good question. We could have a depend on [list of other questions]. This can be handled in code though, so no need to change the model.